blocks/init/src/blocks/custom/columns/columns-block.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-unused-vars, import/no-extraneous-dependencies */
+
+import React from 'react';
+import { Fragment } from '@wordpress/element';
+import { InspectorControls } from '@wordpress/block-editor';
+import { getActions } from '@eightshift/frontend-libs/scripts/editor';
+import manifest from './manifest.json';
+import { ColumnsOptions } from './components/columns-options';
+import { ColumnsEditor } from './components/columns-editor';
+
+export const Columns = (props) => {
```

Comment:
attributes can be destructured here

Reply:
no they can't, if you destructure props you can't use it inside this file and we are using it inside actions helper.